save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/usersettingsview/UserSettingsView.kt

```diff
@@ -282,4 +298,12 @@ abstract class UserSettingsView : AbstractView<UserSettingsProps, UserSettingsVi
         .unsafeMap {
             it.decodeFromJsonString<ImageInfo>()
         }
+
+    @Suppress("TYPE_ALIAS")
+    private suspend fun getOrganizations() = get(
+        "$apiUrl/organizations/by-user/not-deleted",
+        Headers(),
+        responseHandler = ::noopResponseHandler
```

Comment:
Do we need `noopResponseHandler` here? Is there any other handling of request error?

Reply:
Yes, I guess we need this handler. Moreover, we need to implement checks whether user is logged in or not

Reply:
I mean, default behavior is to show error code in a modal window if response is not ok. Are there any reasons to suppress this behavior?

Reply:
Well, as I have said before, either we need to show `please login` modal or we need to suppress it. As @akuleshov7 said, he will add `please login` modal in his PR.